_data/groups.yml

```diff
@@ -0,0 +1,6 @@
+administrators:
+    color: "#FFAA00"
```

Comment:
OCN used `#f89406`, I tried it and it looks fine so we probably want to use that.